query-compiler/query-compiler/tests/snapshots/queries__queries@update-m2m-disconnect.json.snap

```diff
@@ -17,9 +19,7 @@ in let 0$id = mapField id (get 0)
              1$id = mapField id (get 1)
          in execute «DELETE FROM "public"."_CategoryToPost" WHERE
                      ("public"."_CategoryToPost"."B" = ($1) AND
-                     "public"."_CategoryToPost"."A" IN /*
-                     prisma-comma-repeatable-start */$2/*
-                     prisma-comma-repeatable-end */)»
+                     "public"."_CategoryToPost"."A" = $2)»
```

Comment:
This change from `IN [$2]` to `= $2` seems unexpected, the query it refers it is not wrapped in `unique`.

Reply:
Nice catch.

Reply:
Solved